src/pages/publication/submission_tab/submission_description_presentation.tsx

```diff
@@ -0,0 +1,36 @@
+import LabelContentLine from '../../../components/LabelContentLine';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { FormikProps, useFormikContext } from 'formik';
+import { Publication } from '../../../types/publication.types';
+import SubmissionContentText from './submission_content_text';
+
+const SubmissionDescriptionPresentation: React.FC = () => {
+  const { t } = useTranslation('publication');
+  const { values }: FormikProps<Publication> = useFormikContext();
+
+  return (
+    <>
+      <LabelContentLine label={t('description.abstract')}>{values.abstract}</LabelContentLine>
+      <LabelContentLine label={t('description.description')}>{values.description}</LabelContentLine>
+      <LabelContentLine label={t('description.npi_disciplines')}>
+        {values.npiDiscipline.mainDiscipline}
+        {values.npiDiscipline.title && `- ${values.npiDiscipline.title}`}
+      </LabelContentLine>
+      <LabelContentLine label={t('description.tags')}>{`${values.tags.join(', ')}`}</LabelContentLine>
+      <LabelContentLine label={t('common:language')}>{t(`languages:${values.language}`)}</LabelContentLine>
+      <LabelContentLine label={t('description.date_published')}>
+        {values.publicationDate.year}
+        {values.publicationDate.month && `-${values.publicationDate.month}`}
+        {values.publicationDate.day && `-${values.publicationDate.day}`}
```

Comment:
er dette sånn dato skal presenteres? vil det avhenge av hvilket språk brukeren velger? (ikke at det trengs å gjøre noe med det nå, men bare lurer)

Reply:
no idea. Dette ble mer et eksempel på hvordan et "komplekst" objekt kan vises

Reply:
lager en ny task som tar for seg formatering av "komplekse" elementer